bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-dropdown/component.jsx

```diff
@@ -226,7 +226,9 @@ class UserDropdown extends PureComponent {
     const enablePrivateChat = currentUser.isModerator
       ? allowedToChatPrivately
       : allowedToChatPrivately
-      && (!disablePrivChat || (disablePrivChat && hasPrivateChatBetweenUsers(currentUser, user)));
+      && (!(user.locked && disablePrivChat)
```

Comment:
an attendee is able to see the `Chat` option when he click on another attendee, and if he click in the option `akka` eject him

Reply:
Thanks for pointing this out. It should have been `&& (!(currentUser.isLocked && disablePrivChat)`. I confused the two users and didn't realize that they were being turned into mapped users.

Reply:
The check should be good now.